ruff.toml

```diff
@@ -58,7 +58,7 @@ select = [
     "I", # isort
     "UP", # pyupgrade
 ]
-ignore = ["E501","E203","B024","B028","UP037"]
+ignore = ["E501","E203","B024","B028","UP037", "UP035", "UP006"]
```

Comment:
UP006 makes sense to me, we can do a follow up to fix UP035: https://docs.astral.sh/ruff/rules/deprecated-import/